packages/addons/service/dispmanx_vnc/source/system.d/service.system.dispmanx_vnc.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=dispmanx_vnc
+After=kodi.service
+
+[Service]
+ExecStart=/bin/sh /storage/.kodi/addons/service.system.dispmanx_vnc/bin/dispmanx_vncserver-service
```

Comment:
Hi @lrusak 
I moved all pieces of script (to copy the default config, to wait) into dispmanx_vncserver-service shell script. This is also where I intend to collect configuration from the addon, in order to pass it on to the service (see, e.g., inadyn-service). This way, users will be able to configure the service from Kodi, without having to ssh to LE. I hope you agree with this.
Regards, @